[Bazel CI] FATAL: Bazel crashed due to an internal error while evaluating node

CI: https://buildkite.com/bazel/bazel-at-head-plus-downstream/builds/4093#0191abc3-c514-4071-8dda-ff7ad9084d9f

 Platform: Ubuntu

 Logs: 
```

bazel --nosystem_rc --nohome_rc info
FATAL: bazel crashed due to an internal error. Printing stack trace:
java.lang.IllegalStateException: java.lang.RuntimeException: Unrecoverable error while evaluating node 'PlatformMappingValue.Key{path=platform_mappings, wasExplicitlySetByUser=false}' (requested by nodes 'BuildConfigurationKeyValue.Key{buildOptions=1a1e367614cf4c436a40b59b0da811d24bd9acb481ab2d9d559457b7aabecffd}')
	at com.google.devtools.build.lib.skyframe.SkyframeExecutor.evaluateSkyKeys(SkyframeExecutor.java:2030)
	at com.google.devtools.build.lib.skyframe.SkyframeExecutor.evaluateSkyKeys(SkyframeExecutor.java:2006)
	at com.google.devtools.build.lib.skyframe.SkyframeExecutor.createBuildConfigurationKey(SkyframeExecutor.java:1962)
	at com.google.devtools.build.lib.skyframe.SkyframeExecutor.getConfiguration(SkyframeExecutor.java:1886)
	at com.google.devtools.build.lib.runtime.commands.InfoCommand.lambda$exec$0(InfoCommand.java:158)
	at com.google.common.base.Suppliers$NonSerializableMemoizingSupplier.get(Suppliers.java:181)
	at com.google.devtools.build.lib.runtime.commands.InfoCommand.exec(InfoCommand.java:219)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.execExclusively(BlazeCommandDispatcher.java:679)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.exec(BlazeCommandDispatcher.java:252)
	at com.google.devtools.build.lib.server.GrpcServerImpl.executeCommand(GrpcServerImpl.java:607)
	at com.google.devtools.build.lib.server.GrpcServerImpl.lambda$run$1(GrpcServerImpl.java:676)
	at io.grpc.Context$1.run(Context.java:566)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.RuntimeException: Unrecoverable error while evaluating node 'PlatformMappingValue.Key{path=platform_mappings, wasExplicitlySetByUser=false}' (requested by nodes 'BuildConfigurationKeyValue.Key{buildOptions=1a1e367614cf4c436a40b59b0da811d24bd9acb481ab2d9d559457b7aabecffd}')
	at com.google.devtools.build.skyframe.AbstractParallelEvaluator$Evaluate.run(AbstractParallelEvaluator.java:547)
	at com.google.devtools.build.lib.concurrent.AbstractQueueVisitor$WrappedRunnable.run(AbstractQueueVisitor.java:435)
	at java.base/java.util.concurrent.ForkJoinTask$AdaptedRunnableAction.exec(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinPool.scan(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(Unknown Source)
Caused by: java.lang.NullPointerException: Null repoMapping
	at com.google.devtools.build.lib.cmdline.AutoValue_Label_RepoContextImpl.<init>(AutoValue_Label_RepoContextImpl.java:20)
	at com.google.devtools.build.lib.cmdline.Label$RepoContext.of(Label.java:122)
	at com.google.devtools.build.lib.skyframe.config.PlatformMappingFunction.compute(PlatformMappingFunction.java:82)
	at com.google.devtools.build.lib.skyframe.config.PlatformMappingFunction.compute(PlatformMappingFunction.java:60)
	at com.google.devtools.build.skyframe.AbstractParallelEvaluator$Evaluate.run(AbstractParallelEvaluator.java:467)
	... 7 more
``` 
 Culprit: 

Note : This issue is due to https://github.com/bazelbuild/bazel/issues/23023. After --enable_workspace is flipped to false, users have to explicitly specify it where it's needed, or fully migrate to Bzlmod.

 CC Greenteam @fweikert

--- a/.bazelci/presubmit.yml
+++ b/.bazelci/presubmit.yml
@@ -15,6 +15,13 @@ tasks:
       LANG: "C.UTF-8"
     shell_commands:
       - "sudo apt -y update && sudo apt -y install libgmp-dev"
+      # TODO(cb) remove once fully migrated to bzlmod
+      - |
+        echo ${USE_BAZEL_VERSION}
+        if [ $(echo ${USE_BAZEL_VERSION} | cut -d. -f1) != 6 ]; then
+          echo '(enabling workspace mode)' >&2
+          echo 'common --enable_workspace' > .bazelrc.local
+        fi
     build_flags:
       - "--config=ci-common"
       - "--config=linux-bindist"

